analyzer.py

```diff
@@ -175,8 +176,15 @@ def process(self, events):
         # OBJECT SELECTION #
         ####################
 
-        # muon and electron selections are broken out into standalone functions
-        tightElectrons, looseElectrons = selectElectrons(events)
+        coll = CollectObject(events)
+        tightElectrons = coll.Electron({
+            "pt" : (">", 53),
+            "eta" : ("<", 2.4),
+        })
+        looseElectrons = coll.Electron({
+            "pt" : (">", 53),
+            "eta" : ("<", 2.4),
+        })
```

Comment:
Just demonstrating how it could be used,
other id selections have to be implemented here as well

Reply:
if you want more userfirendly approach, we can add
"id" : "tight"
and then in the back in `CollectObject`, we could do something like
```
if id == "tight":
    id_cut = 2
elif id == "loose":
    id_cut = 1
...
mask = electron.id > id_cut
```
but maybe not worth it? not sure